otter/test/test_logging_treq.py

```diff
@@ -338,3 +338,18 @@ def test_delete_timeout(self):
         On timed out call to delete, failure is returned and request logged
         """
         self._test_method_timeout('delete')
+
+    def test_contents(self):
+        """
+        ``content``, ``json_content``, and ``text_content`` are just directly
+        mapped to treq.
+        """
+        ltreq_instance = logging_treq.LoggingTreq()
+        for name in ('content', 'json_content', 'text_content'):
+            for obj in (ltreq_instance, logging_treq):
+                # patching won't change things that were defined at import time
+                expected = (getattr(self.treq, name), getattr(treq, name))
+                actual = getattr(obj, name)
+                self.assertIn(actual, expected,
```

Comment:
I guess this needs deeper understanding of import machinery. If what you are saying is true and is expected behavior as per python spec, then how about checking that `expected` is same object as main `treq`'s functions only?